src/App.css

```diff
@@ -2396,6 +2396,13 @@ code, .code {
   word-break: normal;
 }
 
+@media screen and (min-width: 600px) {
+  .table-view > .children > .child > .thought-container > .thought,
+  .table-view > .children > .child > .thought-container > .thought-annotation {
+    margin-right: 1rem;
+  }
+}
```

Comment:
@incora-dev The root issue was that a style was being applied to `.thought` but not `.thought-annotation`. `.thought-annotation` is the thought overlay that contains the superscript, so it should use the same layout styles as `.thought`.

In this case I decided I liked some margin on larger screens.